### ISSUE TITLE
Add example to vim.org page

### ISSUE THREAD

**Issue by step-** [tpope/vim-surround#169]
You could add this frequent case example involving HTML editing with repeat.vim support:
Change single line to HTML paragraph:
* cursor on line
* `ySS<p>`
* repeat by moving to the next line and pressing `.`

I couldn't achieve the above using line visual mode. I tried `VS<p>`, which worked for the first line but then repeating with `.` produced unexpected results.

Cheers

**Comment by tommcdo** [comment]
Repeating Visual mode commands rarely works as intended. There are infinitely many ways to accomplish any given Visual selection, and Vim has no reasonable way of reproducing your semantic interpretation of it. Instead, it just repeats on the same relative start and end positions.

That said, from what I can tell, surround.vim doesn't hook into repeat.vim for Visual mode operations.

**Comment by step-** [comment]
Exactly. That's what brought me to suggest adding the `ySS<p>` example, or at least mentioning `ySS` in the plugin page on vim.org, which currently says:

    The "." command will work with ds, cs, and yss if you install repeat.vim, vimscript #2136.
